source/Nevermore/Advanced/ReadTransaction.cs

```diff
@@ -26,16 +26,19 @@ namespace Nevermore.Advanced
     public class ReadTransaction : IReadTransaction, ITransactionDiagnostic
     {
         static readonly ILog Log = LogProvider.For<ReadTransaction>();
-        static DbConnection DefaultConnectionFactory(string connectionString) => new SqlConnection(connectionString);
+        protected static DbConnection DefaultConnectionFactory(string connectionString) => new SqlConnection(connectionString);
 
-        readonly RelationalTransactionRegistry registry;
+        readonly IRelationalTransactionRegistry registry;
         readonly RetriableOperation operationsToRetry;
         readonly IRelationalStoreConfiguration configuration;
         readonly ITableAliasGenerator tableAliasGenerator = new TableAliasGenerator();
 
         readonly Func<string, DbConnection> connectionFactory;
+        readonly Action<string>? customCommandTrace;
         readonly string name;
 
+        protected bool TransactionIsNevermoreOwned { get; } = true;
```

Comment:
Trivia: This class is called `ReadTransaction`; the property name `TransactionIsNevermoreOwned` makes it unclear if we're talking about `this` being nevermore owned, or the internal SQL transaction being nevermore owned.

Maybe something like `OwnsSqlTransaction` or perhaps the inverse `IsSqlTransactionExternal` or something?

Reply:
Thanks, I like `OwnsSqlTransaction` so I've gone with that 👍